genesis/generated/901-l2-addresses.json

```diff
@@ -0,0 +1,18 @@
+{
+  "OpChainProxyAdmin": "0xa997d7ebd1700558b8e18a9096fc16f4098dc634",
+  "AddressManager": "0xc3c632e95ae78bb07449a4977ae9025a9258228e",
+  "L1ERC721BridgeProxy": "0x68a7cdc5bbc0146a57cdd18e42c7d17fc3d2c715",
+  "SystemConfigProxy": "0x8e46f8035e7251683cfe4570740b1c8b2f173174",
+  "OptimismMintableERC20FactoryProxy": "0xc56f57ecaa24899d8fd70908f07a2fa70cfb3bcf",
+  "L1StandardBridgeProxy": "0x82233841d0de03e0d7cee1bf06b17f9071578417",
+  "L1CrossDomainMessengerProxy": "0x721214724ad72ff4bb3603a95e041f4ef733365c",
+  "OptimismPortalProxy": "0xc92f7768dd850a850f4512d02840989ab7520920",
+  "DisputeGameFactoryProxy": "0xf7046e2dbbc017d76afaa7c03f87aed547c566c7",
+  "AnchorStateRegistryProxy": "0xc8a07aefdc8efc1545d409ad5517a34fdfb07e57",
+  "AnchorStateRegistryImpl": "0xc77e3e906a1588930d101756b932336384176448",
+  "FaultDisputeGame": "0x0000000000000000000000000000000000000000",
```

Comment:
i'm assuming this is just some config thing in the monorepo making this an empty address

Reply:
yep! the default interopgen recipe doesn't use fault proofs yet